sdk/v2/internal/restmachinery/outbound_request.go

```diff
@@ -0,0 +1,24 @@
+package restmachinery
+
+// OutboundRequest models of an outbound API call.
+type OutboundRequest struct {
+	// Method specifies the HTTP method to be used.
+	Method string
+	// Path specifies a path (relative to the root of the API) to be used.
+	Path string
+	// QueryParams optionally specifies any URL query parameters to be used.
+	QueryParams map[string]string
+	// authHeaders optionally specifies any authentication headers to be used.
```

Comment:
Starting w/ this field and down, the godoc appears to need updating (field needs capitalizing)

Reply:
Nice catch. Fallout from a last-minute refactor.